go.mod

```diff
@@ -9,11 +9,10 @@ require (
 	github.com/onsi/ginkgo v1.10.2
 	github.com/onsi/gomega v1.7.0
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
```

Comment:
related?

Reply:
probably a `make vendor` change, I can remove it